test/index.ts

```diff
@@ -52,7 +52,7 @@ async function runTests() {
         passed,
         channelType,
       };
-      console.log(testResult);
+      console.log(JSON.stringify(testResult));
```

Comment:
This log can be removed again? Keeping it for later tests might not be detrimental though.

Reply:
It was there before, this is just making my ad-hoc test framework ever so slightly less useless.